### PR TITLE
Main adjust features

### PR DESCRIPTION
**Pull request by Spirolife**
This changes a few aesthetic things but also adds feedback when adding new tags to a listing.

### CODE REVIEW

File: Frontend/src/components/Login.js [hunk 1]
```diff
@@ -120,17 +123,35 @@ const Login = ({ setUser }) => {
     position: 'fixed'
   };
 
+  const dmImageStyle = {
```
Comment (pokeblokdude) [inline Frontend/src/components/Login.js:126]: You could just reuse the `bgImage` code by having a ternary opertator where it says `url(${bgImage})`. Doesn't super matter tho

**Comment by pokeblokdude** [comment]
image
I think the white-on-green logo should have a drop shadow or something, idk why but the green-on-gray transition just doesn't look good to me. Also if there's a way to fix it, it's a little annoying that the login card changes in size and position a little bit when you toggle the theme.

**Comment by pokeblokdude** [comment]
Oh and also the biggest problem is that there's currently no way to un-select tags that you've selected by clicking on them directly from a listing.

File: Frontend/src/components/Login.js (hunk 1 above)
Comment (Spirolife) [reply Frontend/src/components/Login.js:126]: Yanno fair. mind changing it? I'm studying for a 311 exam I have at 6 tonight

Comment (pokeblokdude) [reply Frontend/src/components/Login.js:126]: Writing a paper rn but I can later

**Comment by Spirolife** [comment]
> Oh and also the biggest problem is that there's currently no way to un-select tags that you've selected by clicking on them directly from a listing.
> 
> Also something I just noticed is that the background is completely invisible when opening a popup in dark mode

I don't know what you

**Comment by Spirolife** [comment]
> Oh and also the biggest problem is that there's currently no way to un-select tags that you've selected by clicking on them directly from a listing.
> 
> Also something I just noticed is that the background is completely invisible when opening a popup in dark mode

I dont see what you mean by background is invisible

**Review by thomas-hoyt — APPROVED** [review]

**Review by pokeblokdude — APPROVED** [review]

**Pull request merged**